### ISSUE TITLE
TypeScript: Missing `children` on ContainerProperties, and other issues.

### ISSUE THREAD

**Issue by manspaniel** [pmndrs/uikit#46]
When installing components from the `default` kit, the installed components are importing the `ContainerProperties` type from `@react-three/uikit` as props, but that type is missing `children`.

This is leading to TypeScript errors.

I notice that `Container` itself manually adds the `children` prop via:
```
props: { children?: ReactNode } & ContainerProperties & ...etc
```

However in components like `accordion.tsx`, no such children prop is added to the prop types.

Also getting errors on other style properties, like `bottomBorderWidth`.

![CleanShot 2024-04-27 at 22 10 39@2x](https://github.com/pmndrs/uikit/assets/1719776/ae973cbc-e462-401f-a6c6-a781306fc823)

Versions:

```
"@react-three/fiber": "^8.16.2",
"@react-three/uikit": "^0.2.9",
"react": "^18",
```

Are the generated files depending on an unreleased version of uikit, or perhaps the generated files are out of date?

**Comment by bbohlender** [comment]
> Are the generated files depending on an unreleased version of uikit, or perhaps the generated files are out of date?

yes, sorry uikit 0.3.0 is arround the corner, sorry for that
for now you can use the "@rect-three/uikit-default" library which has the old version of the components prebuilt

**Comment by manspaniel** [comment]
Thanks Bela! Looking forward to it 😊

**Comment by bbohlender** [comment]
@manspaniel should be fixed now in 0.3.0 :)
